SettingsInfo: open containing folder on wallet log path click

xiphon opened the pull request:
Partially addresses https://github.com/monero-project/monero-gui/issues/2474

selsta commented:
#2474 asks for opening the wallet folder for the wallet and the log file directly for the log. I would also prefer it this way. Code looks good otherwise.

xiphon commented:
> opening the wallet folder for the wallet 

yep, implemented this as well and updated the PR

> the log file directly for the log

We should open the folder, not the file. There is a plenty of other things (except viewing the log) a typical user would like to do: copy the file, remove it, wipe old logs, etc.
